src/main/scala/com/fulcrumgenomics/util/NumericCounter.scala

```diff
@@ -42,9 +41,13 @@ object NumericCounter {
   * zero for any item that has not been counted yet.  Implements some useful methods
   * to compute statistics when the objects being counted are numeric types.
   */
-class NumericCounter[T](implicit numeric: Numeric[T], tt: TypeTag[T]) extends SimpleCounter[T] {
+class NumericCounter[T](implicit numeric: Numeric[T]) extends SimpleCounter[T] {
+  import scala.collection.JavaConversions._
```

Comment:
Could you import just the 1 or 2 conversions that are needed?  I dislike importing all of them as they can have unintended consequences.